runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientDescribeFactory.java

```diff
@@ -855,7 +858,7 @@ private void doApplicationReset(
             //client.stream = nullIfClosed(state, client.stream);
 
             doReset(application, originId, routedId, initialId, initialSeq, initialAck, initialMax,
-                    traceId, client.authorization, extension);
```

Comment:
The same comment here about conditional sending frame.

Reply:
Yes, agreed - it is already conditional in `doApplicationResetIfNecessary` though.